docs/modules/operation/pages/provisioning/detectors/BsfDetector.adoc

```diff
@@ -0,0 +1,48 @@
+= BSF Detector
+
+The BSF Detector lets you run arbitrary scripts using the Bean Scripting Framework to determine the availability of a service. This can be a script written in any of the languages supported by BSF, the only requirement is that the script returns the string `OK` if it passes.
```

Comment:
```suggestion
The BSF Detector lets you run arbitrary scripts using the Bean Scripting Framework to determine the availability of a service. This can be a script written in any of the languages BSF supports. The only requirement is that the script returns the string `OK` if it passes.
```

Reply:
One sentence per line, please.